app/views/admin/questions/_form.html.erb

```diff
@@ -105,6 +106,20 @@
       }
     });
 
+    $('.cancel-question').on("click", function(event) {
```

Comment:
How about something like 

$('.question').on("click", ".cancel-question", function(event) { ... }

This will then allow for getting the dom_id from the `.question` object.